ui/src/containers/invoices.list/columns.tsx

```diff
@@ -16,82 +13,68 @@ const currencyDecimal = 2;
 
 export const getColumns: (
   opt: getColumnsOptions
-) => ColumnDef<V1Beta1Invoice, any>[] = ({
-  orgMap,
-  isLoading,
-  billingOrgMap,
-}) => {
+) => ApsaraColumnDef<V1Beta1Invoice>[] = ({ orgMap, billingOrgMap }) => {
   return [
     {
       header: "Date",
+      filterVariant: "date",
       accessorKey: "due_date",
-      cell: isLoading
-        ? () => <Skeleton />
-        : ({ row, getValue }) => {
-            const date = getValue() || row.original.period_end_at;
-            return <Text>{dayjs(date).format("MMM DD, YYYY")}</Text>;
-          },
+      cell: ({ row, getValue }) => {
+        const date = getValue() || row.original.period_end_at;
+        return <Text>{dayjs(date).format("MMM DD, YYYY")}</Text>;
```

Comment:
I think we had created a constant somewhere for the date format, right? Can we use that here?